secure-api-gateway-ob-uk-common-datamodel/src/main/java/com/forgerock/sapi/gateway/ob/uk/common/datamodel/converter/v4/common/FRConsentStatusConverter.java

```diff
@@ -41,12 +47,28 @@ public class FRConsentStatusConverter {
         v4tov3ConsentStatus = consentStatusTransalations.entrySet().stream().collect(toMap(Map.Entry::getValue, Map.Entry::getKey));
     }
 
+    static {
+        final Map<String, String> consentStatusTransalations = new HashMap<>();
+        consentStatusTransalations.put(OBDomesticVRPConsentResponseDataStatus.AWAITINGAUTHORISATION.getValue(), "AWAU");
```

Comment:
Not worth import static of `AWAITINGAUTHORISATION`?
(and elsewhere).

Reply:
I have optimized the area around this a bit but it is a bit difficult to make it look good because there are multiple classes (some with the same name but different packages) that contain constants with the same names. I will raise a new issue for this, to either split the conversion in dedicated status convertors for each payment type, or create a general enum that covers all the status values from all payment types.

Reply:
Ok, just a suggestion though. If it complicates things then don't do it.